### ISSUE TITLE
documentation - examples with data

### ISSUE THREAD

**Issue by ori-n** [felixnext/sklearn-recommender#1]
Hi,

I want to try sklearn-recommender and wanted to learn about it by example but could not find any?
Is there an example to share? I'm talking about full flow - data set example, transform, train and predict 

Thanks.

**Comment by felixnext** [comment]
Yeah, the documentation is a bit spotty. I created a ToDo item for me to change that, however it might take a week or two before I come around to do that. In the meantime: [This is a larger project](https://github.com/felixnext/stock_trend_analysis) where I used the library, hope that helps. Other than that feel free to ask questions.
Also you are more then welcome to contribute and create a PR with a rough draft of documentations and examples 😊

**Comment by felixnext** [comment]
Also I derived the implementation from some early work done [here](https://github.com/felixnext/ds_recommender/blob/master/notebooks/Recommendations_with_IBM.ipynb). Maybe that helps as well.
